### DISCUSSION TITLE
windows subsystem for linux

### DISCUSSION THREAD

**Discussion by jscancella** [concourse/concourse#6651] [Help & Support]
Can concourse be run in ubuntu that is running under windows subsystem for linux?

**Comment by vito** [comment]
Yep. This is my development workflow at the moment. Haven't noticed any issues. I run it using `docker-compose`.

**Reply by jscancella** [reply]
Did you have to do anything to set it up? I get the message 
```
The command 'docker-compose' could not be found in this WSL 2 distro.
We recommend to activate the WSL integration in Docker Desktop settings.
```

when I try and run `docker-compose up -d`

**Reply by jscancella** [reply]
FYI for anyone else that ever finds this, yes you have to follow the guide https://docs.docker.com/docker-for-windows/wsl/ that is referenced when you try and run docker inside linux on windows subsystem for linux
